src/latency.c

```diff
@@ -628,6 +768,20 @@ void latencyCommand(client *c) {
                 resets += latencyResetEvent(c->argv[j]->ptr);
             addReplyLongLong(c,resets);
         }
+    } else if (!strcasecmp(c->argv[1]->ptr,"histogram") && c->argc >= 2) {
+        /* LATENCY HISTOGRAM*/
+        if (c->argc == 2) {
+            latencyAllCommandsFillCDFResp(c);
+        } else {
+            latencyCommandFillCDFResp(c);
```

Comment:
maybe rename this to have the word "Single" (where the other one had "all).

Reply:
@oranagra renamed it to `latencySpecificCommandsFillCDFHuman` given it accepts 1 or more commands.